app/main/forms/awards.py

```diff
@@ -0,0 +1,26 @@
+from flask.ext.wtf import Form
+from wtforms import RadioField, BooleanField, validators
+
+
+class AwardedSupplierForm(Form):
+    """Form for the buyer to tell us which supplier was award a contract
+    """
+    # supplier choices expected to be set at runtime
+    supplier = RadioField(
+        "Winning Supplier",
+        validators=[validators.DataRequired(message="You need to answer this question.")],
+        coerce=int
+    )
+
+    def __init__(self, *args, **kwargs):
+        """
+            Requires extra keyword arguments:
+             - `supplier_list` - list of suppliers for the multiple choice
+        """
+        super(AwardedSupplierForm, self).__init__(*args, **kwargs)
+
+        # popping this kwarg so we don't risk it getting fed to wtforms default implementation which might use it
+        # as a data field if there were a name collision
+        suppliers = kwargs.pop("suppliers", [])
+        self.supplier.choices = [(s['id'], s['name']) for s in suppliers]
```

Comment:
This form reads like we are passing a supplier ID and a supplier name. However in reality we are passing in a brief response ID and a supplier name for that brief response. We should make this clear